src/mvBasicWidgets.cpp

```diff
@@ -6220,8 +6252,8 @@ DearPyGui::draw_image_button(ImDrawList* drawlist, mvAppItem& item, mvImageButto
 				texture = static_cast<mvDynamicTexture*>(config.texture.get())->_texture;
 
 			ImGui::PushID(item.uuid);
-			if (ImGui::ImageButton(texture, ImVec2((float)item.config.width, (float)item.config.height),
-				ImVec2(config.uv_min.x, config.uv_min.y), ImVec2(config.uv_max.x, config.uv_max.y), config.framePadding,
+			if (ImGui::ImageButton(std::to_string(item.uuid).c_str(), texture, ImVec2((float)item.config.width, (float)item.config.height),
```

Comment:
Two things here:

1. `std::to_string(item.uuid).c_str()` - I've checked other widgets that need to pass `str_id` to ImGui, and they use `item.info.internalLabel.c_str()`. No need for `std::to_string` here. Moreover, since it does a `PushID`, even a constant empty string would suffice :joy:.

2. `ImGui::ImageButton` no longer accepts frame padding, but we can still support the `frame_padding` argument by doing exactly what the old `ImGui::ImageButton` did:

Before the call to `ImageButton()`:
```cpp
    if (config.framePadding >= 0)
        PushStyleVar(ImGuiStyleVar_FramePadding, ImVec2((float)config.framePadding, (float)config.framePadding));
```
After the call:
```cpp
    if (config.framePadding >= 0)
        PopStyleVar();
```

This way the existing scripts won't break if they use this argument. Let's also declare it as `DEPRECATED_KEYWORD_ARG` so that it spits a warning but still works.